package.json

```
@@ -23,29 +23,29 @@
     "requireindex": "^1.2.0"
   },
   "devDependencies": {
-    "@typescript-eslint/parser": "^4.23.0",
+    "@typescript-eslint/parser": "^4.29.3",
     "all-contributors-cli": "^6.20.0",
     "chai": "^4.3.4",
-    "coveralls": "^3.1.0",
-    "eslint": "^7.26.0",
+    "coveralls": "^3.1.1",
+    "eslint": "^7.32.0",
     "eslint-plugin-eslint-comments": "^3.2.0",
-    "eslint-plugin-eslint-plugin": "^2.3.0",
-    "eslint-plugin-markdown": "^2.1.0",
-    "eslint-plugin-mocha": "^8.1.0",
+    "eslint-plugin-eslint-plugin": "^3.5.3",
+    "eslint-plugin-markdown": "^2.2.0",
+    "eslint-plugin-mocha": "^9.0.0",
     "eslint-plugin-node": "^11.1.0",
     "eslint-plugin-unicorn": "^30.0.0",
-    "markdownlint-cli": "^0.27.1",
-    "mocha": "^8.4.0",
+    "markdownlint-cli": "^0.28.1",
+    "mocha": "^9.1.0",
     "mocha-lcov-reporter": "^1.3.0",
     "npm-run-all": "^4.1.5",
     "nyc": "^15.1.0",
     "outdent": "^0.8.0",
     "semver": "^7.3.4",
     "shelljs": "^0.8.4",
-    "typescript": "^4.2.3"
+    "typescript": "^4.3.5"
   },
   "peerDepencencies": {
-    "eslint": ">=6.0.0 <8.0.0"
+    "eslint": ">=7.0.0 <8.0.0"
```

ESLint 8 beta has been released and eslint-plugin-qunit should be compatible with it so I'd like to support ESLint 8 here: https://eslint.org/blog/2021/08/eslint-v8.0.0-beta.0-released
```suggestion
    "eslint": ">=7.0.0 <9.0.0"
```

---

Expanding the max version of ESLint supported is not a breaking change, and we could do this in a patch or minor release. I would prefer to handle that separately, especially if we want to stick our necks out on the eslint@8.0.0-beta series.

Feel free to create a separate issue to track this, if you like.

---

Fair enough! Fine to wait then.